vlib/vweb/README.md

```diff
@@ -20,8 +20,7 @@ in the browser. No need to quit the app, rebuild it, and refresh the page in the
 
 - **Very fast** performance of C on the web.
 - **Small binary** hello world website is <100 KB.
-- **Easy to deploy** just one binary file that also includes all templates. No need to install any
-  dependencies.
+- **Easy to deploy** just one binary file that also includes all templates. No need to install any dependencies.
```

Comment:
You can wrap the line, so that the 2 pieces individually will be < 100 characters long.